internal/backends/python/pypi_map.override.go

```diff
@@ -3,20 +3,24 @@ package python
 /*
 Manual module -> package mapping overrides
 */
-var moduleToPypiPackageOverride = map[string]string{
-	"grpc_status":  "grpcio-status",       // 2nd most popular
-	"nvd3":         "python-nvd3",         // not popular enough 6th in popularity
-	"requirements": "requirements-parser", // popular rlbot depends on it, but doesn't supply requires_dist
-	"base62":       "pybase62",            // it was overridden by base-62 which wins due to name match but is less popular by far
-	"faiss":        "faiss-cpu",           // faiss is offered as precompiled wheels, faiss-cpu and faiss-gpu.
-	"graphics":     "graphics.py",         // this package is popular, but the module doesn't match the package name https://anh.cs.luc.edu/python/hands-on/3.1/handsonHtml/graphics.html#a-graphics-introduction
-	"replit.ai":    "replit-ai",           // Replit's AI package
-	/* Proxy packages
-	 *
-	 * These are packages that provide helpful aliases, but otherwise provide no functionality.
-	 * We should prefer the real version.
-	 */
-	"discord": "discord.py",
-	"bs4":     "beautifulsoup4",
-	"glm":     "PyGLM",
+var moduleToPypiPackageOverride = map[string][]string{
+	"grpc_status":  {"grpcio-status"},       // 2nd most popular
+	"nvd3":         {"python-nvd3"},         // not popular enough 6th in popularity
+	"requirements": {"requirements-parser"}, // popular rlbot depends on it, but doesn't supply requires_dist
+	"base62":       {"pybase62"},            // it was overridden by base-62 which wins due to name match but is less popular by far
+	"faiss":        {"faiss-cpu"},           // faiss is offered as precompiled wheels, faiss-cpu and faiss-gpu.
+	"graphics":     {"graphics.py"},         // this package is popular, but the module doesn't match the package name https://anh.cs.luc.edu/python/hands-on/3.1/handsonHtml/graphics.html#a-graphics-introduction
+	"replit.ai":    {"replit-ai"},           // Replit's AI package
+	"glm":          {"PyGLM", "glm"},        // Both of these packages are valid, but PyGLM is a library, glm is an executable.
```

Comment:
i'm confused why this mapping is to a slice of strings, should we keep it 1:1 and keep it in the aliases mapping? i see `glm` is an executable, if you have it installed and `import glm` will that result in an error?

Reply:
i don't think it makes sense to include it as an override, overrides should probably be a flat mapping and `glm` specifically is probably an alias instead

Reply:
Not always, consider if the developer of `glm` were to attempt to use Replit to develop the library, upm would suggest installing PyGLM.

Additionally, this opens us up to a world where we can have ranked or confidence-influenced guessing, based on classes or functions imported, not just absolute package names.

Reply:
> Not always, consider if the developer of glm were to attempt to use Replit to develop the library, upm would suggest installing PyGLM.

isn't that what you've just implemented? i don't think that's a good idea - the developer is probably trying to import from their local namespace right? (if we suggest importing `glm` in this case we should absolutely address that 😬)